src/metabase/api/setup.clj

```diff
@@ -162,26 +168,32 @@
              [:dashboard :boolean]
              [:pulse :boolean]
              [:hidden-table :boolean]
-             [:collection :boolean]]]])
+             [:collection :boolean]
+             [:embedded-resource :boolean]]]])
 
 (mu/defn ^:private state-for-checklist :- ChecklistState
   []
-  {:db-type    (mdb/db-type)
-   :hosted?    (premium-features/is-hosted?)
-   :configured {:email (email/email-configured?)
-                :slack (slack/slack-configured?)}
-   :counts     {:user  (t2/count User)
-                :card  (t2/count Card)
-                :table (t2/count Table)}
-   :exists     {:non-sample-db (t2/exists? Database, :is_sample false)
-                :dashboard     (t2/exists? Dashboard)
-                :pulse         (t2/exists? Pulse)
-                :hidden-table  (t2/exists? Table, :visibility_type [:not= nil])
-                :collection    (t2/exists? Collection)
-                :model         (t2/exists? Card :type :model)}})
+  {:db-type                              (mdb/db-type)
+   :hosted?                              (premium-features/is-hosted?)
+   :interested-in-embedding              (not (= (embed.settings/embedding-homepage) :hidden))
+   :embedding-homepage-dismissed-as-done (= (embed.settings/embedding-homepage) :dismissed-done)
+   :configured                           {:email                (email/email-configured?)
+                                          :slack                (slack/slack-configured?)
+                                          :sso                  (google/google-auth-enabled)
```

Comment:
This is not 100% correct, but it's the same logic used in [anon stats](https://github.com/metabase/metabase/blob/master/src/metabase/analytics/stats.clj#L138).
We already plan on asking for that logic to be changed to consider other SSO methods, ideally that change will be reflected/shared here too